test/ServiceBus.Tests/TestStreamProviders/EHStreamProviderWithSlowConsumingPressureDetecting.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Orleans.Providers.Streams.Common;
+using Orleans.Runtime;
+using Orleans.ServiceBus.Providers;
+using Orleans.Streams;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ServiceBus.Tests.TestStreamProviders
+{
+    public class EHStreamProviderWithSlowConsumingPressureDetecting : PersistentStreamProvider<EHStreamProviderWithSlowConsumingPressureDetecting.AdapterFactory>
```

Comment:
Suggest test run against default stream provider, configured to use SlowConsumingPressureMonitor, rather than custom adapter.  That is the preferred user scenario, so the one we should ensure works and is maintained going forward.